### DISCUSSION TITLE
Saral App Usage Observation: Saral Form Register

### DISCUSSION THREAD

**Discussion by karsood** [Sunbird-Saral/Community#13] [Ideas]
Observation: For any kind of scan, an individual sheet is printed, delivered, distributed, filled, and then maintained by the teacher. Managing these loose sheets makes managing the operations difficult.

Impact: In order to retain written record of marks, the teachers end up maintaining their own registers with the data as well. This leads to 2x the amount of work.

Desire: A Saral register which is a booklet which consists of ~100 forms bound together in a way such that the scans also occur easily.

**Comment by surendrasinghs** [comment]
+1 to the idea.
I was wondering if the existing registers will get replaced with new registers (that work with OCR++ / Scan to digitize type of apps).

Is there any help required for this? I believe this can be done once the use-cases are finalised and the user/teacher has space to mention on the sheet/page - "what was it used for?" (say, pages 1-10 for Exam ABC, pages 10-20 for Survey ABC, and so on).

For example, today teachers get blank attendance registers from the market on which they write class-section-year, etc. Sometimes they use a few pages of the register to note other details of the students, such as fees paid, height, weight, etc.

Do share a working draft of a register that you think would work - would be happy to contribute to it
